packages/charting/src/components/DonutChart/Arc/Arc.tsx

```diff
@@ -0,0 +1,50 @@
+import * as React from 'react';
+import * as shape from 'd3-shape';
+import { IArcProps, IArcStyles } from './Arc.types';
+import { classNamesFunction } from 'office-ui-fabric-react/lib/Utilities';
+import { getStyles } from './Arc.styles';
+
+export class Arc extends React.Component<IArcProps, {}> {
+  // tslint:disable:no-any
+  public arc: any;
+
+  constructor(props: IArcProps) {
+    super(props);
+    this.arc = shape.arc();
```

Comment:
While this appears to be constant, it should probably be a prop since it affects rendered output. You could set its value to shape.arc() in `defaultProps`.